packages/mdc-slider/foundation.js

```diff
@@ -753,8 +748,8 @@ class MDCSliderFoundation extends MDCFoundation {
       topLobeHorizontal = localeStringWidth - (2 * digitWidth);
     }
     const extraHorizontalWidth = topLobeHorizontal - 30;
-    let extraHorizontalWidthLeft = extraHorizontalWidth * 3 / 4;
-    let extraHorizontalWidthRight = extraHorizontalWidth / 4;
+    let extraHorizontalWidthLeft = extraHorizontalWidth / 2;
```

Comment:
Can this be further simplified to just use a single variable in this function (e.g. `const extraHorizontalWidthSide = (topLobeHorizontal - 30) / 2`)? It doesn't look like the left/right values are further modified dependent on left/right, and you'd also be able to remove the isRTL block.

Reply:
Done!